scripts/pkg/update-docs_test.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2023 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pkg
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+const version = "v1.11.0"
+
+func Test_translateRelativeLinks(t *testing.T) {
+	tests := []struct {
+		fullPath string
+		md       []byte
+		expected []byte
+	}{
+		{
+			fullPath: "website/content/docs/v1.11.0/somefile.md",
+			md:       []byte("texts[filename](https://github.com/antrea-io/antrea/blob/main/test/test.yaml)texts"),
+			expected: []byte("texts[filename](https://github.com/antrea-io/antrea/blob/main/test/test.yaml)texts"),
+		},
+		{
+			fullPath: "website/content/docs/v1.11.0/somefile.md",
+			md:       []byte("texts[filename](./docs/ci/filename.md)texts"),
+			expected: []byte("texts[filename](./docs/ci/filename.md)texts"),
+		},
+		{
+			fullPath: "website/content/docs/v1.11.0/docs/somefile.md",
+			md:       []byte("texts[filename](build/filename.md)texts"),
+			expected: []byte("texts[filename](build/filename.md)texts"),
+		},
+		{
+			fullPath: "website/content/docs/v1.11.0/docs/somefile.md",
+			md:       []byte("texts[filename](../filename.md)texts"),
+			expected: []byte("texts[filename](../filename.md)texts"),
+		},
+		{
+			fullPath: "website/content/docs/v1.11.0/somefile.md",
+			md:       []byte("texts[filename](build/yamls/filename.yml)texts"),
+			expected: []byte("texts[filename](https://github.com/antrea-io/antrea/blob/v1.11.0/build/yamls/filename.yml)texts"),
+		},
+		{
+			fullPath: "website/content/docs/v1.11.0/somefile.md",
+			md:       []byte("texts[filename](/build/yamls/filename.yaml)texts"),
+			expected: []byte("texts[filename](https://github.com/antrea-io/antrea/blob/v1.11.0/build/yamls/filename.yaml)texts"),
+		},
+		{
+			fullPath: "website/content/docs/v1.11.0/docs/somefile.md",
+			md:       []byte("texts[filename](../build/yamls/filename.yml)texts"),
+			expected: []byte("texts[filename](https://github.com/antrea-io/antrea/blob/v1.11.0/build/yamls/filename.yml)texts"),
+		},
+		{
+			fullPath: "website/content/docs/v1.11.0/docs/maintainers/release.md",
+			md:       []byte("texts[VERSION](../../VERSION)texts"),
+			expected: []byte("texts[VERSION](https://github.com/antrea-io/antrea/blob/v1.11.0/VERSION)texts"),
+		},
+	}
+	for _, tt := range tests {
+		t.Run("", func(t *testing.T) {
```

Comment:
could you try giving an explicit name to each of your subtests?

Reply:
added, but names may not be good enough!